arangod/VocBase/LogicalView.h

```diff
@@ -163,10 +163,26 @@ class DBServerLogicalView : public LogicalView {
     bool isNew
   );
 
+  //////////////////////////////////////////////////////////////////////////////
+  /// @brief fill and return a jSON description of a View object implementation
+  //////////////////////////////////////////////////////////////////////////////
+  virtual void getPropertiesVPack(
```

Comment:
why do we need this function?

Reply:
to fill and return a jSON description of a View object implementation